src/fidesops/schemas/saas/saas_config.py

```diff
@@ -104,6 +104,37 @@ def validate_request_for_pagination(cls, values: Dict[str, Any]) -> Dict[str, An
             pagination_strategy.validate_request(values)
         return values
 
+    @root_validator
+    def validate_grouped_inputs(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        """Validate that grouped_inputs must reference fields from the same collection"""
+        grouped_inputs = values.get("grouped_inputs")
+
+        if grouped_inputs:
+            request_params = values.get("request_params", [])
+            names = [param.name for param in request_params]
+
+            if not all(field in names for field in grouped_inputs):
```

Comment:
If we used sets here we could use a clearer `if not names.issubset(grouped_inputs): ...`